dev/AppLifecycle/ExtensionContract.h

```diff
@@ -25,8 +25,8 @@ namespace winrt::Microsoft::Windows::AppLifecycle::implementation
         { ExtendedActivationKind::File, c_fileContractId, &FileActivatedEventArgs::Deserialize },
         { ExtendedActivationKind::Protocol, c_protocolContractId, &ProtocolActivatedEventArgs::Deserialize },
         { ExtendedActivationKind::StartupTask, c_startupTaskContractId, &StartupActivatedEventArgs::Deserialize },
-        { ExtendedActivationKind::Push, c_pushContractId, &winrt::Microsoft::Windows::PushNotifications::Deserialize },
-        { ExtendedActivationKind::AppNotification, c_appNotificationContractId, &winrt::Microsoft::Windows::PushNotifications::Deserialize },
+        { ExtendedActivationKind::Push, c_pushContractId, &winrt::Microsoft::Windows::PushNotifications::PushDeserialize },
+        { ExtendedActivationKind::AppNotification, c_appNotificationContractId, &winrt::Microsoft::Windows::PushNotifications::ToastDeserialize },
```

Comment:
Its weird to see ToastDeserialize in the PushNotifications namespace rather than AppNotifications

Reply:
Since both were using the same Deserialize function, it was PushNotifications. Since this will be reverted soon I decided to leave it in PushNotifications.